src/vs/workbench/services/search/common/searchExtTypes.ts

```diff
@@ -116,6 +116,11 @@ export interface TextSearchQuery {
 	 * Whether or not to search for whole word matches only.
 	 */
 	isWordMatch?: boolean;
+
+	/**
+	 * Number of threads to use for searching.
+	 */
+	threads?: number;
```

Comment:
Does this get used? I think the `TextSearchOptions` one gets used instead?

Reply:
Removed.